addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -319,7 +319,7 @@ patch(PosStore.prototype, {
         );
     },
     tableHasOrders(table) {
-        return Boolean(table.getOrder());
+        return table.orders.length > 0;
```

Comment:
this doesn't take into account that situation where the table has a parent table, right?

Reply:
That's true. I'll find a different approach to take that into account.